torch/_dynamo/variables/builtin.py

```diff
@@ -581,7 +581,28 @@ def call_function(
             )
         return super().call_function(tx, args, kwargs)
 
-    def _call_min_max(self, tx, a, b):
+    def _call_min_max(self, tx, *args):
+        if len(args) == 1 and args[0].has_unpack_var_sequence(tx):
+            return self._call_min_max_iterable(tx, args[0])
+        elif len(args) == 2:
+            return self._call_min_max_binary(tx, args[0], args[1])
+        else:
+            unimplemented(
+                f"unsupported min / max over {len(args)} args: {[str(a) for a in args]}"
+            )
+
+    def _call_min_max_iterable(self, tx, obj):
+        items = obj.unpack_var_sequence(tx)
+        assert len(items) > 0
+        if len(items) == 1:
+            return items[0]
```

Comment:
You need to propagate guards from obj to this value!

Reply:
It's also done on the return value of the handler:
https://github.com/pytorch/pytorch/blob/fe0e28ab87ddda38a82efe74dd3eaf05e926f117/torch/_dynamo/variables/builtin.py#L563-L565